ds2_configure.py

```diff
@@ -816,7 +816,12 @@ def construct_agent():
             f.write('use_ssl: 1')
 
     logger.exe('cat /var/lib/datastax-agent/conf/address.yaml')
-    logger.exe('sudo chown opscenter-agent:opscenter-agent /var/lib/datastax-agent/conf')
+
+    # pre 5.1
```

Comment:
This relies on the fact that opscenter-agent will only exist for versions of datastax-agent < 5.1.0, the opscenter-agent chown will quietly fail if the user doesn't exist... which is a bit tricky to infer by reading the code. Add a comment here noting that you're being sneaky.

Also, ordering is backwards. Cassandra user should always exist, so put the opscenter-agent chown last.